lib/grp.gi

```diff
@@ -2914,6 +2914,12 @@ function(G,U)
   return U;
 end);
 
+InstallOtherMethod( NormalClosure, "generic method for a list of normal generators",
```

Comment:
This option of calling the method then is undocumented.
And I first was confused by the words "normal generators".

Reply:
Documenting this definitely is a good idea.

Reply:
@PaulaHaehndel done